src/main/play-doc/general/MigrationGuide.md

```diff
@@ -33,18 +33,333 @@ In additional to disabling Elasticsearch via configuration, a service lookup mus
   -Dcontrail.syslog.server.service-locator.enabled=off
 ```
 
-### Service ports
+## Service ports
 
-**The services is now deprecated in lieu of [request acl](RequestAclConfiguration).**
+The services define the protocol, port, and/or path under which your service will be addressed to the outside world on.
 
-The services define the protocol, port, and/or path under which your service will be addressed to the outside world on. For example, if http and port 80 are to be used to provide your services and then the following expression can be used to resolve `/myservice` on:
+**From 1.2 onwards, the service ports are deprecated in lieu of [[request acl|RequestAclConfiguration]].**
 
-```json
-    endpoints        = {
-      "angular-seed-play" = {
-        protocol  = "http"
-        bind-port = 0
-        services  = ["http:/myservice"]
+The [[request acl|RequestAclConfiguration]] in conjunction with [[dynamic proxy configuration|DynamicProxyConfiguration]] provides additional proxy mapping flexibility on top of what was previously provided by the service ports. This will allow operations to have a complete control on the proxy configuration.
+
+### Prerequisite
```

Comment:
Not sure if we need this `Prerequisite` section. This section only tells, that the user should set the `conductrTargetVersion` to 1.2. Once ConductR 1.2 is released we will change `sbt-conductr` so that the `conductrTargetVersion` defaults to 1.2.

If the user need guidance how to use the `conductrTargetVersion` then this guidance should be part of the `sbt-conductr` README.

Therefore I'd remove the `Prerequisite` section here.